packages/augur-state/src/api/Users.ts

```diff
@@ -1,30 +1,222 @@
-// TODO: Define Users class
-
-export interface GetUserTradingPositionsParams {
-  universe: string,
-  creator?: string,
-  category?: string,
-  search?: string,
-  reportingState?: string,
-  disputeWindow?: string,
-  designatedReporter?: string,
-  maxFee?: number,
-  hasOrders?: boolean,
+import {SortLimit} from './types';
+import { DB } from "../db/DB";
+import * as _ from "lodash";
+import { Dictionary, NumericDictionary } from "lodash";
+import { numTicksToTickSize, convertOnChainAmountToDisplayAmount, convertOnChainPriceToDisplayPrice } from "@augurproject/api";
+import { BigNumber } from "bignumber.js";
+import { TrackedUsers } from '../db/TrackedUsers';
+import { ProfitLossChangedLog, OrderFilledLog } from '../logs/types';
+
+export interface UserTradingPositionsParams {
+  universe?: string,
+  account?: string,
+  marketId?: string,
+  outcome?: number,
+};
+
+export interface GetUserTradingPositionsParams extends UserTradingPositionsParams, SortLimit {
+}
+
+export interface GetProfitLossParams {
```

Comment:
This seems like it should extend GetProfitLossSummaryParams to reduce redundancy.